src/main/java/mod/surviving_the_aftermath/event/ForgeEventSubscriber.java

```diff
@@ -33,13 +36,14 @@ public static void raidCompleted(TickEvent.PlayerTickEvent event) {
 		if (player.level() instanceof ServerLevel level) {
 			var raid = level.getRaidAt(player.blockPosition());
 			if (raid != null && raid.isVictory() && !RAIDS.contains(raid.getId())) {
-				var villager = EntityType.VILLAGER.create(level);
-				var allay = EntityType.ALLAY.create(level);
+				var villager = Objects.requireNonNull(EntityType.VILLAGER.create(level));
+				var allay =  Objects.requireNonNull(EntityType.ALLAY.create(level));
```

Comment:
Uncessesary to use requireNonNull for these two lines, we know they will never be null (and even if they are null we will get NPE on the lines below, so no need for requireNonNull

Reply:
IDEA warns are annoyed tbh